src/pages/Patch.tsx

```diff
@@ -42,16 +46,23 @@ const PatchCore: React.FC = () => {
         `There was an error loading the patch: ${e.message}`
       ),
   });
+
   useEffect(() => stopPolling, [stopPolling]);
 
   const patch = get(data, "patch");
   const status = get(patch, "status");
   const description = get(patch, "description");
   const activated = get(patch, "activated");
+
+  const commitQueuePosition = patch?.commitQueuePosition ?? null;
+  const isPatchOnCommitQueue = commitQueuePosition !== null;
+
   if (activated === false) {
```

Comment:
Does this condition need to be adjusted so we don't redirect if `isPatchOnCommitQueue` is true?

Reply:
yeah!